### PR TITLE
config: change 'version' to 'ignitionVersion'

### PR DESCRIPTION
**Pull request by crawford**
This will make it easier to recognize this as an Ignition configuration
file.

### CODE REVIEW

File: doc/ignition.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ following section.
 
 ### Specification ###
 
-- **version** (integer): the version number of the spec. Must be `1`.
+- **ignition_version** (integer): the version number of the spec. Must be `1`.
```
Comment (jonboulle) [inline doc/ignition.md]: camelCase is more idiomatic json fwiw.

**Comment by vcaputo** [comment]
lgtm

**Comment by crawford** [comment]
camelCase for JSON and spaces for YAML

```json
{
    "keyName": "value"
}
```

```yaml
key name: value
```

**Comment by crawford** [comment]
Updated to use camelCase and .... english? /cc @jonboulle

File: doc/ignition.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ following section.
 
 ### Specification ###
 
-- **version** (integer): the version number of the spec. Must be `1`.
+- **ignitionVersion** (integer): the version number of the spec. Must be `1`.
```
Comment (jonboulle) [inline doc/ignition.md]: Somewhere you need to mention this is the only required field and the rest are optional

**Comment by jonboulle** [comment]
> camelCase for JSON and spaces for YAML

whackadoo
LGTM

**Pull request merged**